<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReproVideo</class>
 <widget class="QDialog" name="ReproVideo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reproductor de Video VideoDrake</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Imagenes/multimedia.png</normaloff>:/Imagenes/multimedia.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.000000000000000</double>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="Phonon::VideoPlayer" name="videoPlayer">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>121</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="Phonon::SeekSlider" name="seekSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>485</width>
          <height>23</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="minimumSize">
         <size>
          <width>88</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>122</width>
          <height>23</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
        <property name="numDigits">
         <number>8</number>
        </property>
        <property name="mode">
         <enum>QLCDNumber::Dec</enum>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
        <property name="intValue" stdset="0">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>56</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Play</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/Imagenes/play.png</normaloff>:/Imagenes/play.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>59</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Stop</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/Imagenes/stop1.png</normaloff>:/Imagenes/stop1.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>68</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Pausa</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/Imagenes/pause.png</normaloff>:/Imagenes/pause.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="pushButton_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>59</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Abrir</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/Imagenes/abrir.png</normaloff>:/Imagenes/abrir.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QPushButton" name="pushButton_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>57</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Salir</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/Imagenes/puerta.jpeg</normaloff>:/Imagenes/puerta.jpeg</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QPushButton" name="pushButton_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>159</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string>Pantalla completa</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/Imagenes/tv.png</normaloff>:/Imagenes/tv.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 8pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="Phonon::VolumeSlider" name="volumeSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>251</width>
          <height>21</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Phonon::VideoPlayer</class>
   <extends>QWidget</extends>
   <header location="global">phonon/videoplayer.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
